build/module-builder/package.json

```diff
@@ -34,24 +34,25 @@
     "mkdirp": "^0.5.1",
     "node-sass": "^4.14.1",
     "raw-loader": "^0.5.1",
+    "lodash": "^4.17.19",
     "rimraf": "^2.6.2",
     "sass-loader": "^7.1.0",
     "style-loader": "^0.23.0",
     "tar": "^4.4.19",
     "ts-loader": "^6.0.0",
     "typescript": "^4.4.4",
     "typescript-json-schema": "^0.50.0",
-    "webpack": "^4.20.2",
+    "webpack": "4.46",
```

Comment:
Why use a fixed version?

Reply:
It's the last version of webpack 4, I really don't want to dive into migrating to w5 :P